libero/src/main/groovy/com/connexta/libero/Config.groovy

```diff
@@ -38,8 +45,8 @@ class Config {
         mavenRepo = project.maven.repo
         quickBuild = project.maven.quickbuild
```

Comment:
@lessarderic i'm thinking if we can get #1 merged first i can just move the Options class as part of this pr, since i'll have to resolve some of the differences anyway